charts/sonarqube-dce/README.md

```diff
@@ -16,7 +16,10 @@ Please note that this chart does NOT support SonarQube Community, Developer, and
 
 ## Installing the chart
 
-Please ensure that the value for `ApplicationNodes.jwtSecret` is set with something like `echo -n "your_secret" | openssl dgst -sha256 -hmac "your_key" -binary | base64` and persist this in your `values.yaml`.
```

Comment:
As discussed, we can put a link to the general documentation in this section

Reply:
I just added a note including the link to the general documentation